includes/ets_user_qa_function.php

```diff
@@ -195,176 +197,187 @@ public function ets_ask_qustion_tab() {
 				);
 			?>
 			<?php  
-			}  
-			$loadMoreButtonName = get_option('ets_load_more_button_name');
-			$productQaLength = get_option('ets_product_q_qa_list_length'); 
-			$loadMoreButton = get_option('ets_load_more_button'); 	
-			$pagingType = get_option('ets_product_qa_paging_type' ); 
-			$all_questions = get_post_meta( $productId,'ets_question_answer', true );
+		} 
+		 	
+		echo do_shortcode( '[display_qa_list]' );	
```

Comment:
We created this shortcode, and we are internally using it internally, it does not make sense. Sometimes we may have to use third party shortcode in our code with `do_shortcode` but registering and using the shortcode within the same plugin does not make sense.

Check this: https://chat.openai.com/share/436ae660-af14-4c8c-9cfb-a5654d15a8be


Create a function to do all the common stuff and use the same function inside the method `display_qa_listing_shortcode`